### PR TITLE
Remove duplicate heading, combine Nectar, Bloom and Clover, add Tropiiify

### PR DESCRIPTION
**Pull request by martimpassos**
@mathewjordan and @adamjarling might want to take a look at how I described and where I placed Clover.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -120,7 +120,6 @@ As defined by Wikipedia a shim is a small library that transparently intercepts
 
 - [CanvasPanel](http://canvas-panel.netlify.com/) - React library to build IIIF Presentation 3 level viewing experiences including support for annotations.
 - [Chronoscope World](https://mprove.de/chronoscope/world.html) – A viewer for georeferenced IIIF maps. It is also a fast viewer for browsing IIIF books.
-- [Clover IIIF](https://samvera-labs.github.io/clover-iiif/) - IIIF Presentation API Manifest viewer handling Image, Sound, and Video canvases for React.js
```
Comment (mathewjordan) [inline readme.md:123]: If we can, I think Clover IIIF still belongs under the IIIF Viewers heading. If we need to be specific about which components in Clover are _viewers_, we can say **Clover IIIF Viewer** and link directly to it at https://samvera-labs.github.io/clover-iiif/docs/viewer.

File: readme.md [hunk 2]
```diff
@@ -217,7 +217,6 @@ Tools and resources that provide functionality for presenting IIIF materials in
 
 - [Adno](https://adno.app/en/) - a web application for viewing, editing and sharing narratives and pathways on IIIF images.
 - [Annona Range Storyboard](https://ncsu-libraries.github.io/annona/range/) - [Annona](https://ncsu-libraries.github.io/annona/) toolkit which allows for the guided viewing of segments of a manifest, in addition to the [Annona Multi Storyboard Viewer](https://ncsu-libraries.github.io/annona/multistoryboard/) for guided comparison of multiple manifests.
-- [Bloom IIIF](https://samvera-labs.github.io/bloom-iiif/) - IIIF Presentation API Collection items React.js component that renders a horizontal slider
```
Comment (mathewjordan) [inline readme.md:220]: Totally okay with getting rid of this. Though technically it is now the Slider component at https://samvera-labs.github.io/clover-iiif/docs/slider

File: readme.md [hunk 3]
```diff
@@ -177,7 +176,7 @@ Various tools for working with images such as cropping tools.
 - [jiiify-presentation](https://github.com/ksclarke/jiiify-presentation) - A Java IIIF Presentation library.
 - [Manifesto](https://github.com/UniversalViewer/manifesto) - IIIF Presentation API client and server utility library.
 - [Manifold](https://github.com/UniversalViewer/manifold) - Wraps Manifesto to provide viewer state and related utilities.
-- [Nectar IIIF](https://samvera-labs.github.io/nectar-iiif/) - IIIF Presentation API 3.0 property fluent HTML5 primitives library for React.js
+- [Clover IIIF](https://samvera-labs.github.io/nectar-iiif/) - IIIF front-end React toolkit including a multimedia viewer, image slider, and HTML5 primitives for manifest properties.
```
Comment (mathewjordan) [inline readme.md:179]: 💯

**Review by mathewjordan — CHANGES_REQUESTED** [review]
@martimpassos, see my note about the removal under IIIF Viewers. I still think it be included there in some way.

**Comment by martimpassos** [comment]
So should Clover appear twice in the list (as viewer and as presentation tool) or should I say in the viewer entry that it also encompasses other front-end utilities?

File: readme.md (hunk 1 above)
Comment (adamjarling) [reply readme.md:123]: Agree with @mathewjordan above.  Point directly at the `Viewer` component 👍

**Comment by mathewjordan** [comment]
Hi @martimpassos. Thank you for putting this in.

I understand that multiple lines might be out of sorts for this type of thing, however, the Viewer is just one component for Clover IIIF and I think does belong as its own line item as it is also incorporated into the [IIIF Cookbook viewer matrix](https://iiif.io/api/cookbook/recipe/matrix/).

In the [IIIF](https://github.com/IIIF/awesome-iiif#iiif-viewers) Viewers section, it could be, with the **Viewer** specific description and link to the Viewer component:

```
[Clover IIIF Viewer](https://samvera-labs.github.io/clover-iiif/docs/viewer) - Presentation API Manifest and Collection viewer handling Image, Sound, and Video canvases for React.js
```

Clover IIIF itself is a Presentation API component library or suite and I do think also belongs in the [Presentation API Libraries](https://github.com/IIIF/awesome-iiif#presentation-api-libraries) section with the description you noted:

```
[Clover IIIF](https://samvera-labs.github.io/clover-iiif/) - IIIF front-end React toolkit including a multimedia viewer, image slider, and HTML5 primitives for manifest properties.
```

**Review by mathewjordan — APPROVED** [review]
Thanks @martimpassos 👍

**Comment by mathewjordan** [comment]
@regisrob Might be able to give a final review of things.

**Comment by mathewjordan** [comment]
Thank you @caitlinPerry.

**Pull request merged**